scripts/setup-circleci.sh

```diff
@@ -65,9 +65,12 @@ wget_and_untar https://boostorg.jfrog.io/artifactory/main/release/1.72.0/source/
 wget_and_untar https://github.com/google/snappy/archive/1.1.8.tar.gz snappy &
 wget_and_untar https://github.com/facebook/folly/archive/v2022.03.14.00.tar.gz folly &
 #  wget_and_untar https://github.com/ericniebler/range-v3/archive/0.11.0.tar.gz ranges-v3 &
+wget_and_untar https://archive.apache.org/dist/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz
 
 wait  # For cmake and source downloads to complete.
 
+cp -a hadoop-2.10.1 /usr/local/
```

Comment:
Don't we need some logic below to build and properly install it?

Reply:
No, this is already prebuilt so we just need to copy it onto our path.

Reply:
Pre-built for which architecture? Is it something that's supposed to work across platforms and architectures? If not, it would be a good idea to get the sources and compile, like we do with the other dependencies.

Reply:
@pedroerp This contains jar dependencies so it will work across platform as long as correct java version is installed

Reply:
ah, this is a java library! Never mind then :) How do we communicate with it, btw? Is there a JNI wrapper somewhere?

Reply:
I use HdfsMinicluster class to issue commands to it. I only need the mini cluster implementation from these libraries for testing the hdfs integration. 
The minicluster is essentially a local node made for testing out functionality.